src/routes/network.router.js

```diff
@@ -21,5 +22,56 @@ router.get(
   }
 )
 
+router.get(
+  '/gas/status',
+
+  async (req, res, next) => {
+    const accounts = await res.locals.ocean.accounts.list()
+    var ret = Object()
```

Comment:
more modern doing the same:

```suggestion
    const ret = {}
```

Reply:
also while we're at it, `ret` is kinda cryptic, not clear what that object is supposed to do from its name. Maybe something like `status` is more precise. 

Likewise, this whole endpoint should return something in its response indicating it worked, which should be part of this object here too I guess.

Reply:
fixed with status.

I'm not sure that we should output the faucet result.
After all, the method is /network/gas/status and the output is always the address & balance.

Should we add a new endpoint, called /network/gas/add ?